frontend/src/app/features/settings/device-settings/component/kobo-sync-settings/kobo-sync-settings-component.html

```
@@ -120,6 +120,24 @@ <h3 class="section-title">
               </div>
             </div>
 
+            <div class="setting-item">
+              <div class="setting-info">
+                <div class="setting-label-row">
+                  <span class="setting-label">{{ t('kobo.allowedDeviceIds') }}</span>
+                </div>
+                <input
+                  pInputText
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)